docs/Authoring.md

```diff
@@ -125,6 +125,23 @@ directory by using `Link` metadata to specify file path in output package.
 </ItemGroup>
 ```
 
+To package content v2 files, use `ContentFile` item type. This item type supports 
+`PackageBuildAction`, `PackageCopyToOutput` and `PackageFlatten` metadata. 
```

Comment:
That's fine. But you don't use your `Package` prefix in your sample below. You're using `ContentFileBuildAction` for metadata instead of `PackageBuildAction`. is that a mistake?